docupload/htmlify.py

```diff
@@ -1,23 +1,41 @@
 '''
 HTMLify: Convert any fileformat supported by pandoc to HTML5
 '''
-
+import glob
 import os
 
 import pypandoc
+from bs4 import BeautifulSoup
 
 
 def get_html(doc_file):
     '''Uses pypandoc to convert uploaded file to HTML5'''
 
     tmp_loc = '/tmp/uploaded_' + str(doc_file)
-
+    
     with open(tmp_loc, 'wb') as tmp_file:
         for chunk in doc_file.chunks():
             tmp_file.write(chunk)
-    html = pypandoc.convert(tmp_loc, 'html5')
+    html = pypandoc.convert( 
+        tmp_loc,
+        'html5',
+        extra_args=['--extract-media=']
+        )
+    img_counter = 1
+    soup = BeautifulSoup(html, 'lxml')
+    for img in soup.findAll('img'):
+        img_name = '/media/' + str(doc_file) + '_' +str(img_counter) + '.jpg'
```

Comment:
`str(doc_file)` makes the image name contain the doc file extension too for eg. I get the image name as `/media/test2.docx_1.jpg` which is not desirable. I guess you are too confused about what to do. I will modify it and merge today.